x-pack/plugins/apm/server/plugin.ts

```diff
@@ -49,30 +57,23 @@ export class APMPlugin implements Plugin<APMPluginContract> {
       usageCollection?: UsageCollectionSetup;
     }
   ) {
-    const logger = this.initContext.logger.get('apm');
+    const logger = (this.logger = this.initContext.logger.get('apm'));
```

Comment:
Like below, let's stick to a single assignment:
```suggestion
    this.logger = this.initContext.logger.get('apm');
```